README.md

```diff
@@ -44,6 +45,14 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 - **Issues**: [adobe](https://github.com/search?q=user%3Aadobe+type:issue&state=open&type=Issues), [magento](https://github.com/search?q=user%3Amagento+type:issue&state=open&type=Issues)
 - **Notes**: See more on [official blog](https://community.magento.com/t5/Magento-DevBlog/Squashtoberfest-2019/ba-p/143694) and their [Squashtoberfest repo](https://opensource.adobe.com/squashtoberfest/)
 
+#### **AntiGravity OverUnity Company Inc.**
+
+- **Sag**: T-Shirts, Camping Tents
+- **Requirements**: Create one or more merged pull requests on [AntiGravity OverUnity Website Repo](https://github.com/Antigravity-Overunity-Company-Inc/Homepage), or the
+- **No Signup Neccessary**:Simply include your contact information in the pull requests comments
+- **Issues**: This website needs serious design work! The final product should be entirely browserbased Javascript, HTML and CSS i.e. a static website.
+- **Coding?**: This is mostly a design challenge for people who enjoy writing reponsive designs in CSS and browser-Jascript.
+
```

Comment:
Please stick to the bullet points as listed in the Contributing file (and the other companies on this list)